sregistry/main/s3/delete.py

```diff
@@ -23,8 +23,24 @@ def remove(self, image, force=False):
     try:
         _object = self.bucket.Object(uri)
         _object.load() # this throws an exception if the object does not exist! -> if delete() fails no exception is thrown...
-        _object.delete()
+    
+        # taken from main.registry.delete.remove
+        continue_delete = True
+        if force is False:
+            response = input("Are you sure you want to delete %s?" % uri)
```

Comment:
If you are going to copy paste code, it might be better done as a function that both use.

Reply:
Fixed in my last commit